mysite/models.py

```diff
@@ -1,5 +1,62 @@
 from django.db import models
+from django.conf import settings
 
 
+class Product(models.Model):
+    description = models.TextField()
+    short_description = models.CharField(max_length=50)
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+    category = models.ForeignKey(
+        'Category',
+        on_delete=models.CASCADE
+    )
+    brand = models.ForeignKey(
+        'Brand',
+        on_delete=models.CASCADE
+    )
+    is_pesent = models.BooleanField(default=True)
 
-# Create your models here.
+
+class Brand(models.Model):
+    name = models.CharField(max_length=50)
+    description = models.TextField()
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+
+class Review(models.Model):
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        default=""
```

Comment:
It can't be blank. We can't have a default for the author field

Reply:
done